third_party/open-policy-agent/gatekeeper/helmify/static/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: eraser
+description: A Helm chart for Eraser
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
```

Comment:
we need to update release-manifest so these are generated

https://github.com/open-policy-agent/gatekeeper/blob/master/Makefile#L368-L369

gotcha is helm wants version without v